_includes/site-identifier.html

```diff
@@ -0,0 +1,56 @@
+<div class="usa-identifier">
```

Comment:
11. And under that footer we have the Identifier with updated link text and branding. You can find more information on why you should use it and how on the [Identifier component page](https://designsystem.digital.gov/components/identifier/)

Because we've updated the colors you might see errors about the accessibility of links on a different colored background. The benefit of using a USWDS color in the [overrides](https://github.com/uswds/uswds-sandbox/pull/41/files#diff-855d13bdbc2088ee17d114e9b1eb47f44a04e08107ddbc14d6001d7ffe81be9cR193) is USWDS handles the accessibility of links on any background for you.